### ISSUE TITLE
Image with alpha transparency does not export alpha channel

### ISSUE THREAD

**Issue by TisFoolish** [Yellow-Dog-Man/Resonite-Issues#2103]
### Describe the bug?

When trying to export the metallic map for one of my materials I was unable to get it to export with the alpha channel. Other images with transparency exported just fine.

### To Reproduce

resrec:///U-Foolish/R-9826778C4C08D31D075A9A3FC2BF07BFCC303F35AFADB00EDC22901DBA540F6C This folder has the image

### Expected behavior

The alpha channel to be preserved

### Resonite Version Number

2024.5.24.202

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Index

### Log Files

[CATBED - 2024.5.24.202 - 2024-05-26 13_12_32.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/15448507/CATBED.-.2024.5.24.202.-.2024-05-26.13_12_32.log)

### Additional Context

 
[The resultant export](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/38511571/8dcfdd04-ddb6-483e-a262-4c5a211009d6)

### Reporters

FoolishSnowmew on Discord

**Comment by Frooxius** [comment]
What steps did you use to export the image? You only provided the asset, but not how you exported it.

**Comment by TisFoolish** [comment]
I've added the following to the report



Grab the image from this folder

resrec:///U-Foolish/R-9826778C4C08D31D075A9A3FC2BF07BFCC303F35AFADB00EDC22901DBA540F6C 

Open the Files tab in your dash

Navigate to a folder

Laser grab the image

Push image through the Plus symbol in the Files ui

Export as PNG

**Comment by shiftyscales** [comment]
This seems like it may be the same underlying issue as #1533.

**Comment by TisFoolish** [comment]
Based on what PJB3005 found, the underlying issue of that is that Res is using a really old clipboard format that doesn't support transparency.  That is unrelated as this export never goes through the clipboard. Plus as stated, I can export other images that have transparency just fine. It's just this specific image that is being weird.

**Comment by Frooxius** [comment]
Yeah this is unrelated to the clipboard issue. It's one of the reasons I asked on the replication steps, because there's multiple ways to export an image out of Resonite.
